package.json

```diff
@@ -29,6 +29,12 @@
     "evm:lint": "yarn workspace @buildwithsygma/evm lint",
     "evm:lint:fix": "yarn workspace @buildwithsygma/evm lint:fix",
     "evm:test:unit": "yarn workspace @buildwithsygma/evm test:unit",
+    "bitcoin:build": "yarn workspace @buildwithsygma/bitcoin build:all",
+    "bitcoin:cleanDist": "yarn workspace @buildwithsygma/bitcoin clean",
+    "bitcoin:test": "yarn workspace @buildwithsygma/bitcoin test",
+    "bitcoin:lint": "yarn workspace @buildwithsygma/bitcoin lint",
+    "bitcoin:lint:fix": "yarn workspace @buildwithsygma/bitcoin lint:fix",
+    "clean:all": "yarn workspace @buildwithsygma/core clean; yarn workspace @buildwithsygma/evm clean",
```

Comment:
We already have a command to  "clean" all the packages

Reply:
I think that was renamed, because I just updated this. Will remove this.